### ISSUE TITLE
Case Study 2b ex 3, Rebate forum is missing

### ISSUE THREAD

**Issue by SkillableBB** [MicrosoftLearning/MB-330-Microsoft-Dynamics-365-Supply-Chain-Management#84]
#### Step 7

Description of issue
Rebate forum for editing for customer US-009 is empty. There's marked information that's included up to the point of getting to the rebate form, so unclear if it's an issue within a UI update, or the information existing within the customer's account  but not generating a rebate form. In either instance, Process rebates for payment is incompletable.
 
 
<img width="950" alt="Screenshot 2024-01-08 110918-5ade9a00-a906-444d-a03a-2d132bd5d025" src="https://github.com/MicrosoftLearning/MB-330-Microsoft-Dynamics-365-Supply-Chain-Management/assets/94069201/c8061208-1ff6-4935-a500-42ee09a6ce69">

**Comment by mstall1ngs** [comment]
Hi @SkillableBB,

Thanks for reporting this on GitHub! Module 3 / Case study 2B / Exercise 3. We are reviewing this and will provide an update once a resolution is in place, but it would be helpful if we had additional clarification regarding this issue that you are experiencing. Any additional details that you are able to provide is greatly appreciated!

I was able to Create a trade agreement named Sales price adjustment; generate and process customer rebates; generate rebate claims on new sales orders, however, the rebate agreements display as read-only. I'll review this in more detail and provide an update as soon as a solution is available.

**Comment by mstall1ngs** [comment]
I was able not able to replicate this issue. I was able to generate and process customer rebates; generate rebate claims on new sales orders, to calculate, approve, & process the customer rebate for the invoice. 
![image](https://github.com/MicrosoftLearning/MB-330-Microsoft-Dynamics-365-Supply-Chain-Management/assets/104943975/8c521187-f45a-4cd0-9824-752352f6a260)

After I processed the rebate, the rebate should have a status of Mark. This message shows that the rebate was processed successfully, and the status of the claims has been changed to Mark. 

![image](https://github.com/MicrosoftLearning/MB-330-Microsoft-Dynamics-365-Supply-Chain-Management/assets/104943975/c7615b82-f7bf-4124-8283-749b2f2487e1)

Customer Rebate program ID for customer US-009
![image](https://github.com/MicrosoftLearning/MB-330-Microsoft-Dynamics-365-Supply-Chain-Management/assets/104943975/e5d47e86-241c-4ca9-baca-17aae9e4c72a)
